### DISCUSSION TITLE
Browser-AddIn

### DISCUSSION THREAD

**Discussion by MWL1701** [nextcloud/bookmarks#1668] [Q&A] [answered]
Hi,
welche Browser-Add-Ins funktionieren denn super mit Google Chrome, sodass Bookmarks in beide Richtungen synchronisiert werden?
Ich habe die OwncloudBookmarks und die FreedomMarks probiert. Beide sind leider nicht so wirklich zu gebrauchen.
Haben Sie eine Alternative dafür?
Oder wie bekomme ich die gespeicherten Bookmarks aus der Cloud in meinen Browser?
Ich freue mich über eine Rückmeldung ...
Danke vorab.

**Comment by marcelklehr** [comment, marked as answer]
You can try https://floccus.org/ -- disclaimer: I'm the author :)

**Reply by MWL1701** [reply]
Great! That’s really a good plug-in.
